utils/update_checkout/update-checkout-config.json

```diff
@@ -177,7 +177,7 @@
                 "llbuild": "release/5.7",
                 "swift-tools-support-core": "release/5.7",
                 "swiftpm": "release/5.7",
-                "swift-argument-parser": "1.0.3",
+                "swift-argument-parser": "1.1.2",
```

Comment:
Please revert this part of the change - if this is intended to go into 5.7, let us first settle main and then look at 5.7 (though I suspect that it is a bit too late for that)

Reply:
@compnerd ✅